hathor/p2p/manager.py

```diff
@@ -132,7 +134,22 @@ def __init__(self, reactor: Reactor, my_peer: PeerId, server_factory: 'HathorSer
     def start(self) -> None:
         self.lc_reconnect.start(5, now=False)
         if settings.ENABLE_PEER_WHITELIST:
-            self.wl_reconnect.start(30)
+            self.start_whitelist_reconnect()
+
+    def start_whitelist_reconnect(self) -> None:
+        # The deferred returned by the LoopingCall start method
+        # executes when the looping call stops running
+        # https://docs.twistedmatrix.com/en/stable/api/twisted.internet.task.LoopingCall.html
+        d = self.wl_reconnect.start(30)
+        d.addErrback(self._handle_whitelist_reconnect_err)
+
+    def _handle_whitelist_reconnect_err(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
I'd rename it to `_handle_whitelist_update_error(...)`. What do you think?

Reply:
The same reason as the previous one.

Reply:
Ok. Makes sense.